scripts/sync-from-sunbeam.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+# the directory of the script
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
This script is looking pretty nice.  Just one thing: the shebang says /bin/sh which isn't guaranteed to be bash, but `BASH_SOURCE` is used here (bash-specific).  I'd recommend:

- change the shebang to /usr/bin/bash, /bin/bash, or /usr/bin/env bash (I'm not sure which is most recommended)
- make this script executable (git stores the executable bit of files, so you can be sure it will stay executable for new clones too)
- in the workflow yaml, call this script directly as an executable: `./scripts/sync-from-sunbeam.sh`

Reply:
there are other bashisms in this script too. Unless you really want this script to work under pure `sh`, setting the shebang to bash is the right way to go